src/EditorFeatures/CSharpTest/Completion/CompletionProviders/DeclarationNameCompletionProviderTests.cs

```diff
@@ -1634,7 +1634,7 @@ public void Method()
             await VerifyItemExistsAsync(markup, "nullables");
         }
 
-        [WorkItem(1220195, "https://github.com/dotnet/roslyn/issues/23590")]
+        [WorkItem(1220195, "https://developercommunity2.visualstudio.com/t/Regression-from-1675-Suggested-varia/1220195")]
```

Comment:
How stable are these links?  Would it be more appropriate to mirror the item to github?

Reply:
I don't like mirroring issues like this to GitHub, especially when the issue is currently marked in a way where it'll auto-close once we ship the update. TBH if that was a concern I'd just delete the WorkItem attribute entirely here, since this test isn't for a complicated edge case that needs explanation, but rather us missing a simple edge case to start.